capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -17,15 +19,74 @@ export class WikiResultsService {
     'https://en.wikipedia.org/w/api.php?origin=*&action=parse&page=';
   private static readonly URL_END = '&format=json';
 
-  search(query: string): Observable<WikiSearchResult> {
+  search(query: string): Observable<{title: string, history: string, furtherReading: Map<string, string>}> {
```

Comment:
It might be worth creating a separate interface for this.

Reply:
+1